src/services/UserService.ts

```diff
@@ -11,11 +13,19 @@ const login = async (username: string, password: string): Promise <any> => {
   });
 }
 
-const getProfile = async (): Promise <any>  => {
-    return api({
-      url: "user-profile", 
-      method: "get",
-    });
+const getProfile = async (payload: any): Promise <any>  => {
+  const baseURL = store.getters['user/getInstanceUrl'];
+  const headers = {
+    'Authorization': `Bearer ${payload.token}`,
+    'Content-Type': 'application/json'
+  }
+
+  return client({
+    url: "user-profile",
+    baseURL: `https://${baseURL}.hotwax.io/api/`, 
```

Comment:
We need to check that whether the baseUrl starts with http or not and if yes the don't add anything to baseUrl.